Clsasses/app.rb

```diff
@@ -0,0 +1,173 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'teacher'
+require_relative 'students'
```

Comment:
- [ Required ] I think it would be nice if you fix the error that occurs when I run your application using the `ruby main.rb` command. The error occurs because of line `5` in `app.rb` file which requires the `students.rb` file which does not exist. The file you should import is `student.rb`. Please see the screenshot below and rectify this.

## Example code
```ruby
  require_relative 'student'
```
## Error when running the project
![Screenshot from 2023-11-11 00-34-37](https://github.com/Izebiafe/School-library/assets/16392046/f8dec685-ae61-45b4-bec7-d46f5eee86e8)